.github/workflows/pythonpackage.yml

```diff
@@ -8,7 +8,7 @@ jobs:
     runs-on: ubuntu-latest
     strategy:
       matrix:
-        python-version: [3.7, 3.8, 3.9]
+        python-version: [3.9]
```

Comment:
Thanks for dropping 3.7, but I don't think we should drop 3.8 as well, as that will still be supported for another year at this point.

Reply:
Any thoughts on the "classifiers" list in setup.py?  The value in the original repository is:

   classifiers=["Programming Language :: Python :: 3.6", "Programming Language :: Python :: 3.7"]